README.md

```diff
@@ -120,7 +117,7 @@ what you'll need can be found in the Search UI documentation.
 
 ### Where do I report issues with the Reference UI?
 
-If something is not working as expected, please open an [issue](https://github.com/swiftype/app-search-reference-ui-react/issues/new).
+If something is not working as expected, please open an [issue](https://github.comelastice/app-search-reference-ui-react/issues/new).
```

Comment:
A little typo `https://github.comelastice`

Reply:
Haha whaaaaat that's weird.